### PR TITLE
[HOST REQUEST] [MODULAR] (or something) Imperial Enclave pirates into NRI raiders

### PR DESCRIPTION
**Pull request by Stalkeros2**
## About The Pull Request
Changes Imperial Enclave pirates into an NRI-approved-and-governed-and-issued (and also disguised as deserters but let's be honest nobody would give a damn) raider party.
This is it.
image
image

Starting generic loadout of everyone but the leader:
image

Leader's loadout:
image

A lot of text that no one reads:
image
image

## How This Contributes To The Skyrat Roleplay Experience
Lore being a part of the game good.
Fulfilling host requests good.
(Totally not fake and this is not me trying to be funny this is me trying not to be lying.)
image

## Changelog

:cl: Stalkeros
add: NRI raiders
del: Imperial Enclave pirates
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/encounters/code/enclave_pirates.dm [hunk 1]
```diff
@@ -1,86 +1,88 @@
...
 	glasses = /obj/item/clothing/glasses/sunglasses
 	ears = /obj/item/radio/headset/guild
+	mask = /obj/item/clothing/mask/gas/hecu2
 
-	uniform = /obj/item/clothing/under/enclave
-	suit = /obj/item/clothing/suit/armor/vest/alt
-	suit_store = /obj/item/gun/ballistic/automatic/m16
+	uniform = /obj/item/clothing/under/costume/nri
+	suit = /obj/item/clothing/suit/armor/vest/russian
+	suit_store = /obj/item/gun/ballistic/automatic/akm
```
Comment (Zonespace27) [inline modular_skyrat/modules/encounters/code/enclave_pirates.dm]: NRI footsoldiers don't get ammo for the AK, so they're stuck with just the magazine from what i understand?

Comment (Stalkeros2) [reply modular_skyrat/modules/encounters/code/enclave_pirates.dm]: I thought I've fixed this fuck.

Comment (Stalkeros2) [reply modular_skyrat/modules/encounters/code/enclave_pirates.dm]: They do, actually - their belt subtype has all the needed ammo.

**Comment by Stalkeros2** [comment]
Even though it's near-completely done and completely ready to merge, I'd really ask for anyone who'd merge that to merge #15159 beforehand so I can add some flags onto the ship later.

**Comment by Stalkeros2** [comment]
(I also honestly don't know if it's the right use of [HOST REQUEST]? I mean, Xyel asked me this? Do I even use it in this case and stuff?)

**Comment by Stalkeros2** [comment]
(Bad deploys can cope and seethe.)

File: modular_skyrat/modules/encounters/code/enclave_pirates.dm [hunk 2]
```diff
@@ -166,11 +166,11 @@
...
+	<br> By the Supreme command, a special meeting of representatives from the Imperial Academy of Finances and the Collegias of Foreign and Internal Affairs, Economy, Defense was convened under the chairmanship of Adjutant General Tarkhanov to consider the issue of the incongruity with the Imperial regulations by the Nanotrasen Research Station.
 	<br> This meeting, having familiarized itself with all the other possible actions and solutions, came to the conviction that the indenture of fines has casus belli to perform a strategic secret operation.
 	<br> The Imperial Regulation has to be enforced in order to minimise any potential threat for the whole Empire, not excluding allied kingdoms, organisations and other partners, and to strengthen our positions in the ongoing Border War.
 	<br>
-	<br> About such a Supreme Will, reported in the recall of the Councillor of the Defense Colleague, No. 217648, We announce to the military department for immediate actions in appropriate cases.
+	<br> About such a Supreme Will, reported in the recall of the Councillor of the Defense Collegium, No. 217648, We announce to the military department for immediate actions in appropriate cases.
```
Comment (Jolly-66) [inline modular_skyrat/modules/encounters/code/enclave_pirates.dm]: The big paper refactor is merged upstream and mirrored here, I would recommend looking at it to retrofit your code ahead of time.

https://github.com/Skyrat-SS13/Skyrat-tg/pull/15362

File: modular_skyrat/modules/encounters/code/enclave_pirates.dm [hunk 3]
```diff
@@ -1,85 +1,89 @@
...
-	rank = "Imperial Enclave Trooper"
-	you_are_text = "You are an Imperial Enclave outfit."
-	flavour_text = "The station has refused to pay the fine for breaking Imperial regulations, you are here to recover the debt. Do so by ransoming crew and stealing credits."
+	rank = "NRI Marine"
+	you_are_text = "You are a Novaya Rossiyskaya Imperiya task force."
+	flavour_text = "The station has refused to pay the fine for breaking Imperial regulations, you are here to recover the debt. Do so by demanding the funds. Force approach is usually recommended, but isn't the only method."
+	important_text = "Allowed races are humans, felinids, Akulas, IPCs. Follow your field officer's orders."
 	spawner_job_path = /datum/job/space_pirate
-	restricted_species = list(/datum/species/human)
+	restricted_species = list(/datum/species/human, /datum/species/human/felinid, /datum/species/akula, /datum/species/robotic/ipc)
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/encounters/code/enclave_pirates.dm]: remove felinid

File: modular_skyrat/modules/encounters/code/enclave_pirates.dm (hunk 2 above)
Comment (Stalkeros2) [reply modular_skyrat/modules/encounters/code/enclave_pirates.dm]: :ok_hand: Gotcha.

**Comment by Stalkeros2** [comment]
As mentioned in one of the reviews above, it'd be a good idea to merge this after #15362 to prevent conflicts.

**Comment by Lukaster42** [comment]
AK-25 is way too shitty and Krinkov is way too good in it's current state, i suggest to give them new ERT Submachine Gun.

**Comment by Gandalf2k15** [comment]
fix issues

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**